src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -33,8 +34,10 @@ public void Flush ()
 				zip.Close ();
 				zip.Dispose ();
 				zip = null;
+				//File.Copy (archive, $"{flush}_{archive}", overwrite: true);
```

Comment:
Why is this line commented out?

Reply:
I need to re-write it as it broke the build. It was just really late last night when I saw the errors so it was quicker to just comment out for now.